pkg/kubectl/categories/categories.go

```diff
@@ -21,10 +21,15 @@ import (
 	"k8s.io/client-go/discovery"
 )
 
+// CategoryExpander interface provides a higher layer of group resource mapping.
```

Comment:
Maybe even shorter: CategoryExpander maps category strings to GroupResouces. Categories are classification or 'tag' of a group of resources.